### DISCUSSION TITLE
大量数据快速刷新，会报错，正常吗？

### DISCUSSION THREAD

**Discussion by goto-art** [XCharts-Team/XCharts#218] [Q&A]
用的2.40版本，同时刷新10个图表，其中有5个折线图，刷新间隔为0.02s，数据量约2000条，会报许多除数为零的错，我看了下，大致是在自动分轴的时候，是我哪里设置错误的问题吗？
![Snipaste_2022-07-31_00-35-39](https://user-images.githubusercontent.com/69996452/181933436-1890e37b-9373-4210-896f-45c5cae33f6d.png)

**Comment by monitor1394** [comment]
splitNumber是怎么设置的？看报错是splitNumber计算出来是0了。

**Reply by goto-art** [reply]
splitNumber设置的是10，Large 阈值设置的是1000，我也奇怪为什么会计算成0,。数据增加的频率快（0.01s），每次都刷新图表，这点有影响吗？

**Reply by monitor1394** [reply]
本地调试下看看是哪里的问题？

**Reply by monitor1394** [reply]
要具体看代码是怎么写的
